src/connector/src/parser/debezium/avro_parser.rs

```diff
@@ -25,14 +25,13 @@ use risingwave_common::error::ErrorCode::{InternalError, ProtocolError};
 use risingwave_common::error::{Result, RwError};
 use risingwave_pb::plan_common::ColumnDesc;
 
-use super::operators::*;
 use crate::impl_common_parser_logic;
-use crate::parser::avro::util::{
-    avro_field_to_column_desc, extract_inner_field_schema, from_avro_value,
-    get_field_from_avro_value,
-};
+use crate::parser::avro::util::avro_field_to_column_desc;
 use crate::parser::schema_registry::{extract_schema_id, Client};
 use crate::parser::schema_resolver::ConfluentSchemaResolver;
+use crate::parser::unified::avro::{AvroAccess, AvroParseOptions};
+use crate::parser::unified::debezium::DebeziumAdapter;
+use crate::parser::unified::util::apply_row_operation_on_stream_chunk_writer;
 use crate::parser::util::get_kafka_topic;
 use crate::parser::{SourceStreamChunkRowWriter, WriteGuard};
 use crate::source::{SourceColumnDesc, SourceContextRef};
```

Comment:
Line 46:

```
// TODO: avoid duplicated codes with `AvroParser`
```

Is this completed in this PR?

Reply:
Maybe. I found the duplicated codes for parsing is done in this PR. But there are still some duplicated codes for handling schema fetching.